draft-ietf-webpush-protocol.xml

```diff
@@ -1284,6 +1292,38 @@ HEADERS      [stream 82] +END_STREAM
           </list>
         </t>
       </section>
+      <section title="Service Name and Port Number Registration" anchor="iana.port">
+        <t>
+          <list style="hanging">
+            <t hangText="Service Name.">
+              <vspace />
+              webpush</t>
+            <t hangText="Transport Protocol.">
+              <vspace />
+              tcp</t>
+            <t hangText="Assignee.">
+              <vspace />
+              IESG (iesg@ietf.org)
+            </t>
+            <t hangText="Contact.">
+              <vspace />
+              IETF Chair (chair@ietf.org)

```

Comment:
I would identify the webpush@ietf.org group first, with the chair as a fallback.